NuGet.config

```diff
@@ -16,6 +16,7 @@
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
     <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
     <add key="dotnet5-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5-transport/nuget/v3/index.json" />
+    <add key="dotnet6-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6-transport/nuget/v3/index.json" />
```

Comment:
I don't think we can add this feed to release/5.0. We probably need to branch icu for 5.0 too and make it publish to dotnet5-transport.

Reply:
is this necessary?

Reply:
Yep, let's stay with dotnet5 feed

Reply:
Yeah I agree we should branch off to release/5.0 and publish to the 5.0 channel on ICU. That way if we need to service ICU for 5.0 we do it there.

Reply:
we created a branch for release/5.0 on dotnet/icu now.